Source/IntegrationTests/TestFiles/LitTests/LitTest/blogposts/TestGenerationNoInliningEnumerativeDefinitions.dfy

```diff
@@ -1,11 +1,11 @@
 
 // Generating and Running Block-Based Tests:
 // RUN: %baredafny generate-tests %args Block %S/TestGenerationNoInliningEnumerativeDefinitions.dfy > %t-tests.dfy
-// RUN: %baredafny test %args --unicode-char:false --target:cs "%t-tests.dfy" >> "%t"
+// RUN: %baredafny test %args --allow-deprecation --unicode-char false --target:cs "%t-tests.dfy" >> "%t"
```

Comment:
@Dargones how come the test generation blogpost is using `--unicode-char false`?

Reply:
#5013 should fix this, among other things, since I have updated the format in which counterexamples report unicode characters. Once #5013 is merged, it should be possible to `--allow-deprecation --unicode-char false` from these tests as well

Reply:
Thanks, I'll remove unicode-char=false here then.